spec/classes/apache_spec.rb

```diff
@@ -6,10 +6,10 @@
       {
         id: 'root',
         kernel: 'Linux',
-        lsbdistcodename: 'squeeze',
+        lsbdistcodename: 'jessie',
```

Comment:
oh the fact changes will cause a lot of conflicts during the merge of this + the ubuntu PR :(

Reply:
It will, but it'll be worth it! I had to rebase after the concat_dir param was removed so I've been through it once before at least :P